x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -321,25 +317,9 @@ export function TransactionsTable({
           >
             <p>
               <FormattedMessage
-                id="xpack.apm.transactionsTable.cardinalityWarning.body"
-                defaultMessage="The number of unique transaction names exceeds the configured value of {bucketSize}. Try reconfiguring your agents to group similar transactions or increase the value of {codeBlock}"
-                values={{
-                  bucketSize,
-                  codeBlock: (
-                    <EuiCode>xpack.apm.ui.transactionGroupBucketSize</EuiCode>
-                  ),
-                }}
+                id="xpack.apm.transactionsTable.transactiongrouplimit.exceeded"
+                defaultMessage="The number of unique transaction names limit was exceeded. Try narrowing down your results using the search bar."
```

Comment:
This error message is correct and will be displayed when the Kibana limit (10.000) is reached. 
Similar to what @achyutjhunjhunwala did in https://github.com/elastic/kibana/pull/148641 we may want to also display a warning when the APM Server limit is reached (detected by the presence of the `_other` bucket)

Reply:
- When the Kibana limit (10.000) is reached

<img width="1913" alt="image" src="https://user-images.githubusercontent.com/1313018/213155566-15e94dc8-167d-49ab-b0d4-3d7c95052714.png">

- When we only have `_other` bucket

<img width="1906" alt="image" src="https://user-images.githubusercontent.com/1313018/213155939-fd4e8bbd-9638-4151-a1d3-fc9640cdfaed.png">

Reply:
Could we have a custom label for `_other`? Something like `Other transactions`.